package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "webpack-demo",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "start": "webpack --mode development --watch",
```

Comment:
[ OPTIONAL]
- [ ] You could also use this command to automatically served and opened in your default browser. `"start": "webpack serve --mode development --open",`. I personally prefer this command because I do not have to use LiveServer to do the serving again. It pretty much does it all for you.